sqlmesh/core/console.py

```diff
@@ -2027,6 +2042,25 @@ def show_linter_violations(
             self.log_warning(msg)
 
 
+def _compare_df_cells(x: t.Any, y: t.Any) -> bool:
```

Comment:
[Nit] consider naming this differently to reflect its purpose better, e.g. `_cells_match` (so you'd flip the return values to reflect this).

Should be a low-lift to make it so you read L1911 and understand what's going on without having to look at this.